internal/indexcoord/meta_table.go

```diff
@@ -905,6 +936,10 @@ func (mt *metaTable) RemoveIndex(collID, indexID UniqueID) error {
 	}
 
 	delete(mt.collectionIndexes[collID], indexID)
+	if len(mt.collectionIndexes[collID]) == 0 {
+		delete(mt.collectionIndexes, collID)
+		metrics.IndexCoordIndexTaskNum.DeleteLabelValues(strconv.FormatInt(collID, 10))
```

Comment:
Here, should we reset the guage to zero?

Reply:
there is no longer a chance to clean up these labels, and the metrics data collected by prometheus will get bigger and bigger.